core/rpc/client/gateway/http/client.go

```diff
@@ -120,20 +120,20 @@ func (g *GatewayHttpClient) GetAuthParameter(ctx context.Context) (*types.Gatewa
 		return nil, err
 	}
 
-	if r.Error != "" {
-		return nil, errors.New(r.Error)
+	if r.Message != "" {
+		return nil, errors.New(r.Message)
 	}
 
 	return r.Result, nil
 }
 
 // gatewayAuthPostResponse defines the response of POST request for /auth
 type gatewayAuthPostResponse struct {
-	Error string `json:"error"`
+	Message string `json:"message"` // compatible with swagger format
```

Comment:
More specifically, `google.golang.org/genproto/googleapis/rpc/status.Status`, right?  Is this so the following works?

```go
func parseErrorResponse(respTxt []byte) error {
	// NOTE: here directly use status.Status from googleapis/rpc/status
	var res status.Status
	err := json.Unmarshal(respTxt, &res)
```

Reply:
yeah, If kgw returned filed using `message`, we won't need to change `parseErrorResponse` and it works with kgw.

Reply:
Oh I see the comment is a bit misleading